ui/apps/platform/src/services/ComplianceEnhancedService.ts

```diff
@@ -452,6 +452,12 @@ export function deleteScanConfig(scanConfigId: string) {
     });
 }
 
+export function runComplianceScanConfiguration(scanConfigId: string) {
+    return axios.post<Empty | Error>(`${scanScheduleUrl}/${scanConfigId}/run`).then((response) => {
```

Comment:
Is `| Error` needed?
* `.then(() => {` above implies success has `Empty` as data.
* `.catch((error) => {` above does not need `Error` as an alternative in response type.

Reply:
Odd, because TS complained when I needed the `| Error` when I added the DELETE endpoint above, which is why I did the same thing here. Now, if I remove it in both places, TS is not complaining.